docs/quick-start.md

```diff
@@ -5,11 +5,15 @@ Install Numaflow and run a couple of example pipelines.
 ## Prerequisites
 
 A Kubernetes cluster is needed to try out Numaflow. A simple way to create a local cluster is using Docker Desktop.
-* https://www.docker.com/
-* https://www.docker.com/blog/how-kubernetes-works-under-the-hood-with-docker-desktop/
+<ul>
+  <li><a href="https://www.docker.com/">Docker</a></li>
```

Comment:
won't `[text](url)` style work?, something like
* [foo](foo.example.com)
* [bar](bar.example.com)

Reply:
unfortunately no..

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/19543684/191133582-a8c68b9a-bbd7-4f04-b57a-250f277b98ae.png">
<img width="755" alt="image" src="https://user-images.githubusercontent.com/19543684/191133617-32db816f-430f-4790-a5b4-99298cd5b5b2.png">

Reply:
Should not use HTML tags in the markdown.

Reply:
Should we abandon the list style?

Reply:
discussed offline..
thank you @whynowy for the tips

need one more blank line to render the markdown style list